api/proto/teleport/legacy/types/types.proto

```diff
@@ -3265,10 +3265,9 @@ message LoginStatus {
     (gogoproto.nullable) = false,
     (gogoproto.jsontag) = "lock_expires,omitempty"
   ];
-  // RecoveryAttemptLockExpires contains the time when this lock will expire
-  // from reaching MaxAccountRecoveryAttempts. This field is used to determine
-  // if a user got locked from recovery attempts.
-  google.protobuf.Timestamp RecoveryAttemptLockExpires = 5 [
+  // DEPRECATED RecoveryAttemptLockExpires contained the time when this lock will expire
```

Comment:
You're going to want to use reserved fields for this: https://protobuf.dev/programming-guides/proto3/#fieldreserved

Reply:
Adding to what Zac said, but without looking at the PR in its entirety:

* If you still need to keep the field around, use the [deprecated=true] field option. Some folks add a "// Deprecated: \<reason\>" comment at the end of the field docs, in Go style, but that's optional.

* If you don't need to access the field, you can reserve both its tag (number) and name, effectively removing it from the proto.

Reply:
Thank you for the advice, since we don't need to reference the field I have updated it to be `reserved`